src/__tests__/dummyLogin.js

```diff
@@ -0,0 +1,28 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import server from '../index';
+
+chai.use(chaiHttp);
+
+const { assert } = chai;
+
+describe('Login', () => {
+  it('Should generate Token', async () => {
+    const res = await chai
+      .request(server)
+      .post('/api/v1/auth/login')
+      .send({ email: 'abc123@gmail.com' });
```

Comment:
why should it generate token if no password was given?

Reply:
This is a dummy login route for the purpose of generating token. This will be removed once the login functionality has been implemented